src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/__init__.py

```diff
@@ -12,19 +12,7 @@
 
 def load_params(_):
     import azure.cli.command_modules.batch._params  # pylint: disable=redefined-outer-name, unused-variable
-    try:
-        import azure.cli.command_modules.batch_extensions._params
-    except ImportError as exp:
-        # Optional Batch Extensions not installed
-        logger.debug("Did not load Batch Extensions module")
-        logger.debug(str(exp))
 
 
 def load_commands():
     import azure.cli.command_modules.batch.commands  # pylint: disable=redefined-outer-name, unused-variable
```

Comment:
@derekbekoe - I'm correct in assuming that the new extension model no longer requires these imports, right? :)

Reply:
Correct